### PR TITLE
Update arcade

### LINKED ISSUE #2281: Use new Arcade SDK publishing infrastructure
This repository uses a very old form of the Arcade SDK publishing infrastructure. That infra is planned to be removed from the Arcade SDK by the end of the first half of January 2020. The repository build definition(s) need to be updated to use the new publishing infrastructure, otherwise its official builds will fail to publish. It’s a simple process, [documentation about it is here](https://github.com/dotnet/arcade/pull/4556). If you have any question or problem please reach out to @JohnTortugo, @riarenas, @mmitche or .NET Core Engineering

Once we remove the old infrastructure you might not be able to get an updated Arcade SDK (or Arcade updates at all) if you’re not using the new publishing infrastructure.

### PR DESCRIPTION
**Pull request by nohwnd**
## Description
Update Arcade to the latest version

## Related issue
Fixes #2281

### CODE REVIEW

**Comment by nohwnd** [comment]
There are many changes in this PR because the eng folder was copied from the arcade source, the relevant changes are at the bottom: 

image

File: global.json [hunk 1]
```diff
@@ -1,9 +1,9 @@
 {
   "tools": {
-    "dotnet": "2.2.104"
+    "dotnet": "3.1.100"
   },
   "msbuild-sdks": {
-    "Microsoft.DotNet.Build.Tasks.Feed": "2.2.0-beta.19151.1",
-	"Microsoft.DotNet.Arcade.Sdk": "1.0.0-beta.19320.1"
+    "Microsoft.DotNet.Arcade.Sdk": "5.0.0-beta.20052.1",
+    "Microsoft.DotNet.Helix.Sdk": "5.0.0-beta.20052.1"
```
Comment (singhsarab) [inline global.json]: This is an interesting bump for a version.
@JohnTortugo Can you please help us understand why that is?

Comment (singhsarab) [reply global.json]: Are we aligning with .Net 5 ?

Comment (nohwnd) [reply global.json]: I asked about that: "The update to 5 was due to the release of .net 3.1"

**Review by singhsarab — APPROVED** [review]
:shipit:

**Comment by ViktorHofer** [comment]
Does that mean that the vstest repo is now publishing via arcade to the corresponding channel? Asking as I added a manual nuget push step in the official build defintion in devdiv.

**Comment by nohwnd** [comment]
It does not, I only updated the scripts and packages to the latest arcade version. And then made sure that the build is still able to finish. No big changes to the build were done.

**Comment by ViktorHofer** [comment]
Got it, thanks. Is it planned to switch to the common arcade based publishing infrastructure?

**Pull request merged**